liquibase-package-manager.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: liquibase-package-manager
+  version: "0.2.9"
+  epoch: 0
+  description: "Easily manage external dependencies for Database Development. Search for, install, and uninstall liquibase drivers, extensions, and utilities."
+  copyright:
+    - license: Apache-2.0 AND MPL-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/liquibase/liquibase-package-manager
+      tag: v${{package.version}}
+      expected-commit: 1cd39ecef00dee2b6cb800acd2d2410f06179231
+
+  - uses: go/build
+    with:
+      packages: ./cmd/lpm/darwin.go
+      output: lpm
+
+  - runs: |
+      mkdir -p "${{targets.contextdir}}/bin"
+      ln -sf /usr/bin/lpm ${{targets.contextdir}}/bin/lpm
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: liquibase/liquibase-package-manager
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - go
+        - curl
+  pipeline:
+    - uses: git-checkout
+      with:
+        repository: https://github.com/liquibase/liquibase-package-manager
+        tag: v${{package.version}}
+        expected-commit: 1cd39ecef00dee2b6cb800acd2d2410f06179231
```

Comment:
this looks good but this brings will fail package update on every update. As of now, the bot only updates the git commit inside main pipeline.

we can consider using the following in here after including `git` inside packages above. 
```bash
git clone -b v${{package.version}} https://github.com/liquibase/liquibase-package-manager
```